src/client/containers/FavoritePage/FavoritePage.Container.js

```diff
@@ -0,0 +1,78 @@
+
+import './FavoritePage.styles.css';
+
+   
+import React, { useState, useEffect, useCallback } from "react";
+import FavouriteList from '../../components/FavouriteList/FavouriteList.component';
+
+
+
+export const addProductToFavorites = (user_id,product_id) => {
+  console.log("in add favorites container");
+  console.log(user_id,product_id)
+ 
+  const requestOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        product_id:product_id,
+        user_id: user_id
+        
+       }),
+       
+    };
+    fetch("/api/favorites/add", requestOptions)
+      .then((response) => response.json())
+      .then((result) => {
+        console.log(result);
+      });
+  };
+
+export default function FavoritePage  () {
+  
+  
+  const [favorites, setFavorites] = useState([]);
+
+
+  const GetFavorites = useCallback(() => {
+    let apiUrl = "/api/users/2/favorites";
```

Comment:
I understand this is hard coded because of testing purpose but you should change it to be dynamic and accept any incoming id.